java-checks/src/test/java/org/sonar/java/checks/StandardCharsetsConstantsCheckTest.java

```diff
@@ -21,36 +21,42 @@
 
 import org.junit.jupiter.api.Test;
 import org.sonar.java.checks.verifier.CheckVerifier;
+import org.sonar.java.checks.verifier.internal.InternalCheckVerifier;
 
-import static org.sonar.java.checks.verifier.TestUtils.testSourcesPath;
+import static org.sonar.java.checks.verifier.TestUtils.mainCodeSourcesPath;;
 
 class StandardCharsetsConstantsCheckTest {
 
   @Test
   void test() {
-    CheckVerifier.newVerifier()
-      .onFile(testSourcesPath("checks/StandardCharsetsConstantsCheck.java"))
+    InternalCheckVerifier.newInstance()
+      .onFile(mainCodeSourcesPath("checks/StandardCharsetsConstantsCheck.java"))
       .withCheck(new StandardCharsetsConstantsCheck())
+      .withQuickFixes()
       .verifyIssues();
-    CheckVerifier.newVerifier()
-      .onFile(testSourcesPath("checks/StandardCharsetsConstantsCheck.java"))
+    InternalCheckVerifier.newInstance()
+      .onFile(mainCodeSourcesPath("checks/StandardCharsetsConstantsCheck.java"))
       .withCheck(new StandardCharsetsConstantsCheck())
       .withJavaVersion(7)
+      .withQuickFixes()
       .verifyIssues();
-    CheckVerifier.newVerifier()
-      .onFile(testSourcesPath("checks/StandardCharsetsConstantsCheck_java8.java"))
+    InternalCheckVerifier.newInstance()
+      .onFile(mainCodeSourcesPath("checks/StandardCharsetsConstantsCheck_java8.java"))
       .withCheck(new StandardCharsetsConstantsCheck())
       .withJavaVersion(8)
+      .withQuickFixes()
```

Comment:
I am not sure anymore: Is it necessary to add `withQuickFixes` here because the check now supports quick fixes even though we did not modify the test file?